apps/frontend/src/hooks/useWalletConnect.ts

```diff
@@ -46,6 +46,8 @@ export const useWalletConnect = () => {
 
   const account = useMemo(() => wallets[0]?.accounts[0]?.address, [wallets]);
 
+  const getWallet = useCallback((index: number) => wallets[index], [wallets]);
```

Comment:
You don't need it, we have `useAccount` hook.

Reply:
Hook takes first/default account, this one allows to select additional ones - which will be useful for cross-chain stuff